doc/make_external_gallery.py

```diff
@@ -150,6 +150,24 @@ def format(self):
         link="https://github.com/JacobBumgarner/grad-descent-visualizer",
         image="grad_descent_visualizer.gif",
     ),
+    nikolov1=Example(
+        title="Python Libraries for Mesh, Point Cloud, and Data Visualization (Part 1)",
+        description="Python Libraries for Mesh, Point Cloud, and Data Visualization (Part 1)",
+        link="https://medium.com/towards-data-science/python-libraries-for-mesh-and-point-cloud-visualization-part-1-daa2af36de30",
+        image="nikolov1.gif",
+    ),
+    nikolov2=Example(
+        title="How to Voxelize Meshes and Point Clouds in Python",
+        description="How to Voxelize Meshes and Point Clouds in Python",
+        link="https://medium.com/towards-data-science/how-to-voxelize-meshes-and-point-clouds-in-python-ca94d403f81d",
+        image="nikolov2.gif",
+    ),
+    nikolov3=Example(
+        title="Neighborhood Analysis, KD-Trees, and Octrees for Meshes and Point Clouds in Python",
+        description="Neighborhood Analysis, KD-Trees, and Octrees for Meshes and Point Clouds in Python",
+        link="https://medium.com/towards-data-science/neighborhood-analysis-kd-trees-and-octrees-for-meshes-and-point-clouds-in-python-19fa96527b77",
+        image="nikolov3.gif",
+    ),
```

Comment:
Two things come to mind:
1. It would be nice to have a combination of short title + longer description as in the existing examples (instead of copying the same for both fields), but I think the new website doesn't actually use the description field anywhere (@akaszynski  please correct me if I'm wrong)
2. Since the "title" is what gets rendered in the gallery, I'd consider trying to make these shorter. At least look at the built docs (I haven't, yet) an check that the text doesn't look weird or broken in the gallery.

Reply:
How about following the format of a scientific paper and titling it "Ivan Nikolov, Apr 26, 2022", "Ivan Nikolov, May 31, 2022" and "Ivan Nikolov, Jun 16, 2022"? (I am not a scientist, so my writing style may be wrong.)

Reply:
If we go that route I'd prefer to say something about the contents. So something like
1. Nikolov on Visualization Libraries
2. Nikolov on Voxelization and Point Clouds
3. Nikolov on Neighbourhood Analysis

If "Ivan Nikolov" fits too, that would be even better.

Reply:
Fixed in 659f3da288310efe95bd05662b5b5b22187d7311 .